### DISCUSSION TITLE
Download file with chunks very slow

### DISCUSSION THREAD

**Discussion by noinhotnn** [seaweedfs/seaweedfs#3213] [Q&A]
i have a test. 
With text file 1MB
Situation 1: First upload without 'append' (once) then download speed is very fast (10ms)
Situation 2: I split the file into many small chunks (1KB) and then append. The speed very slow (900ms)
So what is difference here and how to increase speed with file have a lot of chunks

**Comment by chrislusf** [comment]
need to buffer and append large chunks instead of small chunks.

**Reply by noinhotnn** [reply]
Thanks for the quick reply. As i described in #2876.  Data to be read in real time. Is there any way to do it in weed server.
